packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/Suggestions.types.ts

```diff
@@ -12,7 +12,7 @@ export interface ISuggestionsCoreProps<T> extends React.Props<any> {
   /**
    * How the suggestion should look in the suggestion list.
    */
-  onRenderSuggestion?: (props: T, suggestionItemProps: T) => JSX.Element;
+  onRenderSuggestion?: (props: T, suggestionItemProps: any) => JSX.Element;
```

Comment:
Sorry for the delay, do you mind explaining this type change to someone with zero context? Hoping to merge this with confidence tomorrow.

Reply:
Yea I'm not sure I understand why this is changed to any. Can you elaborate?